Steer/DigitizerWorkflow/src/TOFDigitizerSpec.cxx

```diff
@@ -48,6 +49,7 @@ class TOFDPLDigitizerTask : public o2::base::BaseDPLDigitizer
 {
  public:
   TOFDPLDigitizerTask(bool useCCDB, std::string ccdb_url, int timestamp) : mUseCCDB{useCCDB}, mCCDBurl(ccdb_url), mTimestamp(timestamp), o2::base::BaseDPLDigitizer(o2::base::InitServices::FIELD | o2::base::InitServices::GEOM), mPass("unanchored"){};
+  // mPass("unanchored") to be replaced with mPass(o2::conf::DigiParams::Instance().passName) once ccdb ready
```

Comment:
I think you could do this immediately and this would be backward compatible. (I am also not sure this will ever come from CCDB.)